package.json

```diff
@@ -30,22 +30,22 @@
     "lint-staged": "^13.1.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.46.1",
-    "@typescript-eslint/parser": "^5.46.1",
-    "eslint": "^8.29.0",
+    "@typescript-eslint/eslint-plugin": "^6.9.1",
+    "@typescript-eslint/parser": "^6.9.1",
+    "eslint": "^8.53.0",
     "eslint-config-next": "^14.0.1",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-prettier": "^5.0.1",
     "eslint-plugin-react": "^7.33.2",
-    "eslint-plugin-security": "^1.5.0",
-    "eslint-plugin-storybook": "^0.6.8",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "prettier": "^2.8.1"
+    "eslint-plugin-security": "^1.7.1",
+    "eslint-plugin-storybook": "^0.6.15",
+    "eslint-plugin-unused-imports": "^3.0.0",
+    "prettier": "^3.0.3"
   },
   "peerDependencies": {
-    "eslint": "^8.29.0",
-    "prettier": "^2.8.1"
+    "eslint": ">=8.29.0",
+    "prettier": ">=2.8.1"
```

Comment:
@DanWebb I think we'll need to update these peerDeps too, prettier is going to have to be > 3 at least.